testData/solutions/win32.json5

```diff
@@ -5797,6 +6462,498 @@
+                    "SwapMouseButtonsConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.fallbackValue",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.SwapMouseButtons",
+                            "outputPath": "value",
+                            "defaultValue": 0
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": {
+                                    "get": "pvParam",
+                                    "set": "uiParam"
+                                }
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.SwapMouseButtons": "SwapMouseButtonsConfig.value"
+                }
+            },
+            "configureScrollWheelMode": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETWHEELSCROLLLINES",
+                    "setAction": "SPI_SETWHEELSCROLLLINES",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "UINT"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "ScrollWheelModeConfig": {
+                        "schema": {
+                            "title": "Mouse scroll wheel mode",
+                            "description": "Changes the way scroll wheel behaves, between lines and screens",
+                            "type": "number",
+                            "default": 3,
+                            "min": 0,
+                            "max": 25,
+                            "divisibleBy": 1,
+                            // This special value trigger the special meaning
+                            // of full-page scrolling.
+                            "specialValue": "'full-page' | 4294967295"
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "ScrollWheelModeConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollLines",
+                            "defaultOutputPath": "value",
+                            "match": {
+                                "full-page": 4294967295,
```

Comment:
Since this file is now JSON5, you can add a comment to explain the significance of these strange values